### PR TITLE
Remove task_report and git_report

### PR DESCRIPTION
**Pull request by LucasG0**

### CODE REVIEW

File: backend/infrahub/git/integrator.py [hunk 1]
```diff
@@ -214,7 +214,7 @@ async def import_jinja2_transforms(
         local_transforms: dict[str, InfrahubRepositoryJinja2] = {}
 
         # Process the list of local Jinja2 Transforms to organize them by name
-        await self.log.info(f"Found {len(config_file.jinja2_transforms)} Jinja2 transforms in the repository")
+        log.info(f"Found {len(config_file.jinja2_transforms)} Jinja2 transforms in the repository")
```
Comment (dgarros) [inline backend/infrahub/git/integrator.py:217]: FYI, if we are using the standard logger, the log will not show up in prefect
Considering the current size of this PR I think we should revisit this part later but at some point we'll need to use Prefect logger

File: backend/infrahub/message_bus/operations/requests/artifact_definition.py [hunk 2]
```diff
@@ -15,125 +15,120 @@
...
 async def check(message: messages.RequestArtifactDefinitionCheck, service: InfrahubServices) -> None:
-    async with service.task_report(
-        title=f"Artifact Definition: {message.artifact_definition.definition_name}",
-        related_node=message.proposed_change,
-    ) as task_report:
-        log.info(
-            "Validating generation of artifacts",
-            artifact_definition=message.artifact_definition.definition_id,
-            source_branch=message.source_branch,
+    log.info(
```
Comment (dgarros) [inline backend/infrahub/message_bus/operations/requests/artifact_definition.py]: we can remove this log and convert it to the flow_run_name

File: backend/infrahub/message_bus/operations/requests/repository.py [hunk 3]
```diff
@@ -99,43 +99,35 @@ async def checks(message: messages.RequestRepositoryChecks, service: InfrahubSer
...
-                    repository_name=repository.name.value,
-                    commit=repository.commit.value,
-                    file_path=check_definition.file_path.value,
-                    class_name=check_definition.class_name.value,
-                    branch_name=message.source_branch,
-                    proposed_change=message.proposed_change,
-                    branch_diff=message.branch_diff,
-                )
-            )
+    log.info(
```
Comment (dgarros) [inline backend/infrahub/message_bus/operations/requests/repository.py]: Remove this log and add a flow_run_name

File: backend/infrahub/message_bus/operations/requests/proposed_change.py [hunk 4]
```diff
@@ -75,373 +75,326 @@ def log_line(self) -> str:
...
 async def schema_integrity(
     message: messages.RequestProposedChangeSchemaIntegrity,
     service: InfrahubServices,  # pylint: disable=unused-argument
 ) -> None:
-    async with service.task_report(
-        related_node=message.proposed_change,
-        title="Schema Integrity",
-    ):
-        log.info(f"Got a request to process schema integrity defined in proposed_change: {message.proposed_change}")
+    log.info(f"Got a request to process schema integrity defined in proposed_change: {message.proposed_change}")
```
Comment (dgarros) [inline backend/infrahub/message_bus/operations/requests/proposed_change.py]: remove this log and add a flow_run_name

File: backend/infrahub/message_bus/operations/requests/proposed_change.py [hunk 5]
```diff
@@ -75,373 +75,326 @@ def log_line(self) -> str:
...
-    ) as task_report:
-        log.info(f"Refreshing artifacts for change_proposal={message.proposed_change}")
-        definition_information = await service.client.execute_graphql(
-            query=GATHER_ARTIFACT_DEFINITIONS,
-            branch_name=message.source_branch,
-        )
-        artifact_definitions = _parse_artifact_definitions(
-            definitions=definition_information[InfrahubKind.ARTIFACTDEFINITION]["edges"]
-        )
+    log.info(f"Refreshing artifacts for change_proposal={message.proposed_change}")
```
Comment (dgarros) [inline backend/infrahub/message_bus/operations/requests/proposed_change.py]: remove this log and add a flow_run_name

File: backend/infrahub/message_bus/operations/requests/proposed_change.py [hunk 6]
```diff
@@ -545,76 +497,67 @@ async def run_generators(message: messages.RequestProposedChangeRunGenerators, s
...
-                        f"--infrahub-repo-config={config_file}",
-                        f"--infrahub-address={config.SETTINGS.main.internal_address}",
-                        "-qqqq",
-                        "-s",
-                    ],
-                    plugins=[
-                        InfrahubBackendPlugin(service.client.config, repository.repository_id, proposed_change.id)
-                    ],
-                )
+    log.info("running_repository_tests", proposed_change=message.proposed_change)
```
Comment (dgarros) [inline backend/infrahub/message_bus/operations/requests/proposed_change.py]: Convert log to flow_run_name

**Review by dgarros — COMMENTED** [review]
Looks good overall, it's a big one
if it takes too long to get all the tests to pass, it might be easier to break it down into smaller PRs ... up to you

File: backend/infrahub/git/integrator.py (hunk 1 above)
Comment (LucasG0) [reply backend/infrahub/git/integrator.py:217]: Right, I think we would need a `get_log` utility function that returns either server logger or prefect logger depending on the context.

**Pull request merged**